task-sdk/tests/task_sdk/execution_time/test_task_runner.py

```diff
@@ -2647,7 +2647,7 @@ def test_task_runner_both_callbacks_have_timing_info(self, create_runtime_ti):
         """Test that both success and failure callbacks receive accurate timing information."""
         import time
 
-        from airflow.exceptions import AirflowException
+        from airflow.sdk.exceptions import AirflowException
```

Comment:
This would likely be a case where should just raise `RuntimeException` in the FailureOperator -- AirflowException is not special

Reply:
I have left the case of raising runtimeException in this PR and hope to do it in a follow up after looking closely to breaking changes